.ci/Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM ubuntu:16.04
+
+ENV HADOOP_HOME /usr/local/hadoop
+ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
+
+# install packages
+RUN \
+  apt-get update && apt-get install -y \
+  ssh \
+  rsync \
+  vim \
+  openjdk-8-jdk
+
+# download and extract hadoop, set JAVA_HOME in hadoop-env.sh, update path
+RUN \
+  wget http://apache.mirrors.tds.net/hadoop/common/hadoop-3.1.2/hadoop-3.1.2.tar.gz && \
+  tar -xzf hadoop-3.1.2.tar.gz && \
+  rm -rf hadoop-3.1.2.tar.gz && \
+  mv hadoop-3.1.2 /usr/local && \
+  ln -sf /usr/local/hadoop-3.1.2/ /usr/local/hadoop
+
+ADD configs/hadoop_config /usr/local/hadoop
+CMD cat /usr/local/hadoop/hadoop_config >> ~/.bashrc
+
+CMD bash ~/.bashrc
+  ##echo "export JAVA_HOME=$JAVA_HOME" >> $HADOOP_HOME/etc/hadoop/hadoop-env.sh && \
+  ##echo "PATH=$PATH:$HADOOP_HOME/bin" >> ~/.bashrc
+
+##ssh setup
+##RUN \
+##    mkdir ~/.ssh \
+##    chmod 700 ~/.ssh
+
+CMD bash cat "" >> ~/.ssh/authorized_keys \
+    chmod 600 ~/.ssh/authorized_keys
+
+## ssh key gen
+RUN \
+    ssh-keygen -t rsa -P "" -f ~/.ssh/id_rsa
```

Comment:
You don't need this, since you don't need to create an ssh key for the root account within the container

Reply:
okay :+1: